src/redux/user/kesselActions.ts

```
@@ -0,0 +1,33 @@
+/**
+ * Load integrations permissions from Kessel v2 and map them to the existing Redux state structure.
+ * This allows v2 orgs to use Kessel permissions for integrations without changing the rest of the application.
+ *
+ * Note: Sources permissions continue using Chrome API v1 (loaded separately) until sources service migrates to Kessel.
+ *
+ * @param kesselPermissions - Permissions from KesselRbacAccessContext
+ */
+export const loadPermissionsFromKessel =
+  (
+    kesselPermissions: KesselRbacAccessContextValue['permissions']
+  ): ThunkAction<void, any, unknown, AnyAction> =>
```

ESLint

[error] Replace `⏎····kesselPermissions:·KesselRbacAccessContextValue['permissions']⏎··` with `kesselPermissions:·KesselRbacAccessContextValue['permissions']`

(prettier/prettier)